changelogs/fragments/20240411-remove-deprecated-sort_json_policy_dict.yml

```diff
@@ -0,0 +1,5 @@
+---
+deprecated_features:
+  - module_utils.policy - deprecation of the function ``sort_json_policy_dict()`` (https://github.com/ansible-collections/amazon.aws/pull/2052).
```

Comment:
This isn't a deprecation, this is a removal.  The deprecation has already happened.